File: scratch/check_db.js
const mysql = require('mysql2/promise');

async function run() {
    try {
        const connection = await mysql.createConnection({
            host: 'localhost',
            user: 'root',
            password: 'Hemasrikotha@07',
            database: 'smart_medistock'
        });

        const [alerts] = await connection.query('SELECT * FROM inventory_alerts');
        console.log("=== ALL ALERTS ===");
        console.log(JSON.stringify(alerts, null, 2));

        const [medicines] = await connection.query('SELECT * FROM medicines LIMIT 10');
        console.log("=== FIRST 10 MEDICINES ===");
        console.log(JSON.stringify(medicines, null, 2));

        await connection.end();
    } catch (err) {
        console.error("Database connection error:", err);
    }
}

run();

File: scratch/check_db.py
import mysql.connector

try:
    conn = mysql.connector.connect(
        host="localhost",
        user="root",
        password="Hemasrikotha@07",
        database="smart_medistock"
    )
    cursor = conn.cursor(dictionary=True)
    
    cursor.execute("SELECT * FROM inventory_alerts")
    alerts = cursor.fetchall()
    print("ALL ALERTS:")
    for a in alerts:
        print(a)
        
    cursor.close()
    conn.close()
except Exception as e:
    print("Error:", e)
